README.md

```diff
@@ -21,6 +21,8 @@ This server just runs other labrad servers, allowing you to start and stop them
 By running node servers on one or more machines connected to labrad, you can remotely control which labrad servers are running on those machines.
 This can be very useful in distributed setups.
 For more information, see the [source](https://github.com/labrad/pylabrad/blob/master/labrad/node/__init__.py).
+To configure the node (e.g., define directories containing servers), it is necessary to set several registry keys;
+see the [node docstring](https://github.com/labrad/pylabrad/blob/master/labrad/node/__init__.py) for details.
```

Comment:
This seems redundant with the previous sentence which also points to the node source. I would probably suggest deleting the previous sentence "For more information...".